### ISSUE TITLE
Disqus comments isn't working when using HTTPS

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/site#896]
For example, https://software-carpentry.org/blog/2015/03/whats-in-your-bag.html doesn't load. Tested with clean Firefox 36.

This is probably due the script that loads Disqus comments:

~~~
   (function() {
      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
      dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
~~~

The source probably need to be HTTP**S** and not HTTP.

**Comment by ocefpaf** [comment]
@r-gaia-cs I use a protocol agnostic approach in my blog

```javascript
dsq.src = "//" + disqus_shortname + '.disqus.com/embed.js';
```

That should work for both HTTP and HTTPS.

**Comment by r-gaia-cs** [comment]
@ocefpaf Thanks for your comment. I'm trying to fix another bug so if you could send a pull request to this I will be grateful. You can assigned the pull request to Greg review and merge.

**Comment by ocefpaf** [comment]
Sure, forking it now...

**Comment by r-gaia-cs** [comment]
@ocefpaf could you also address https://github.com/swcarpentry/training-course/issues/384 and use the [same solution](https://github.com/swcarpentry/training-course/issues/384#issuecomment-85075173) for our site? Thanks.

**Comment by wking** [comment]
On Mon, Mar 23, 2015 at 09:07:13AM -0700, Filipe wrote:
> @r-gaia-cs I use a protocol agnostic approach in my blog
> 
> ```javascript
> dsq.src = "//" + disqus_shortname + '.disqus.com/embed.js';
> ```
> 
> That should work for both HTTP and HTTPS.

We had been doing this same thing in workshop-template's Eventbrite
link, but it was giving us problems on recent versions of Firefox.
See swcarpentry/workshop-template#153.  I'd second Paul Irish in
encouraging folks to just use https:// everywhere [1,2].

[1]: http://www.paulirish.com/2010/the-protocol-relative-url/
[2]: https://github.com/swcarpentry/workshop-template/pull/153#issuecomment-73544582

**Comment by ocefpaf** [comment]
Done #897 and done https://github.com/swcarpentry/training-course/pull/385.

**Comment by jduckles** [comment]
Looks like this was fixed in #897 closing
